secant/Features/BalanceBreakdown/BalanceBreakdownStore.swift

```diff
@@ -23,7 +23,7 @@ struct BalanceBreakdownReducer: ReducerProtocol {
         var transparentBalance: Balance
         
         var totalBalance: Zatoshi {
```

Comment:
This is confusing. What's this variable supposed to show? We must probably name it something else like `totalSpendableBalance` or something similar

Reply:
`totalSpendableBalance` used, good point